zebra-state/src/service/finalized_state/zebra_db/shielded.rs

```diff
@@ -338,4 +338,22 @@ impl DiskWriteBatch {
 
         self.prepare_history_batch(db, finalized)
     }
+
+    /// Deletes the Sapling note commitment tree at the given [`Height`].
+    pub fn delete_sapling_tree(&mut self, zebra_db: &ZebraDb, height: &Height) {
+        let sapling_tree_cf = zebra_db
+            .db
+            .cf_handle("sapling_note_commitment_tree")
+            .unwrap();
+        self.zs_delete(&sapling_tree_cf, height);
+    }
+
+    /// Deletes the Orchard note commitment tree at the given [`Height`].
+    pub fn delete_orchard_tree(&mut self, zebra_db: &ZebraDb, height: &Height) {
+        let orchard_tree_cf = zebra_db
+            .db
+            .cf_handle("orchard_note_commitment_tree")
+            .unwrap();
+        self.zs_delete(&orchard_tree_cf, height);
+    }
```

Comment:
Optional: Add methods for getting the cf handles.

```suggestion
    /// Deletes the Sapling note commitment tree at the given [`Height`].
    pub fn delete_sapling_tree(&mut self, zebra_db: &ZebraDb, height: &Height) {
        self.zs_delete(&zebra_db.sapling_tree_cf(), height);
    }

    /// Deletes the Orchard note commitment tree at the given [`Height`].
    pub fn delete_orchard_tree(&mut self, zebra_db: &ZebraDb, height: &Height) {
        self.zs_delete(&zebra_db.orchard_tree_cf(), height);
    }
```

Reply:
Oh, nice. I think we can do that in other places as well.

Reply:
> Oh, nice. I think we can do that in other places as well.

This came out of mild disappointment seeing the column names defined in several places - I think we should use constants there too. 

Let's merge this PR and create a ticket for that cleanup?